lib/ansible/plugins/lookup/conjur_variable.py

```diff
@@ -0,0 +1,148 @@
+# (c) 2017, Jason Vanderhoof <jason.vanderhoof@cyberark.com>, Oren Ben Meir <oren.benmeir@cyberark.com>
+# (c) 2017 Ansible Project
```

Comment:
It seems `# (c) 2017 Ansible Project` line should be removed (see: https://github.com/ansible/ansible/pull/34704#issuecomment-356657729).